src/components/stepper-item/stepper-item.tsx

```diff
@@ -135,7 +133,13 @@ export class StepperItem implements InteractiveComponent {
     return (
       <Host aria-expanded={this.active.toString()} onClick={() => this.emitRequestedItem()}>
         <div class="container">
-          <div class="stepper-item-header">
+          <div
+            class="stepper-item-header"
+            tabIndex={
+              /* additional tab index logic needed because of display: contents */
```

Comment:
This part is a bit weird...

In the horizontal case, the content isn't within the "button" whereas in the vertical case, the content is within the "button". That seems like kind of a big difference. Can we make them consistent? The content is within the trigger area in one layout and not in the other.

Reply:
Might be a breaking change but worth it.

Reply:
Sorry for the delayed reply. I agree that both layouts having different structure is odd. In the meantime, I think we should look into addressing the immediate issue and we can discuss redesigning this. cc @macandcheese @ashetland